pybind11_stubgen/parser/mixins/fix.py

```diff
@@ -7,7 +7,9 @@
 import sys
 import types
 from logging import getLogger
-from typing import Any
+from typing import Any, Sequence, TypeVar
+
+from typing_extensions import TypeGuard
```

Comment:
It looks like `CI / Runs on 🐍 3.8-3.11 • gemmi` all failed bc of missing `typing_extensions`... 

Are you willing to make `typing_extensions` a required dependency of this package? Or what is the right solution here?

Reply:
Let's revert to the original code and add `#type: ignore[...]` if necessary.

Reply:
done!